PurchasesCoreSwift/Logging/Strings/Strings.swift

```diff
@@ -1,18 +1,27 @@
 //
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
 // Created by Andrés Boedo on 9/14/20.
-// Copyright (c) 2020 Purchases. All rights reserved.
 //
 
 import Foundation
 
-@objc(RCStrings) public class Strings: NSObject {
-    @objc public static let attribution = AttributionStrings()
-    @objc public static let configure = ConfigureStrings()
-    static let identity = IdentityStrings()
-    static let network = NetworkStrings()
-    static let offering = OfferingStrings()
-    static let purchase = PurchaseStrings()
-    static let purchaserInfo = PurchaserInfoStrings()
-    @objc public static let receipt = ReceiptStrings()
-    static let restore = RestoreStrings()
+enum Strings {
+
+    static let attribution = AttributionStrings.self
+    static let configure = ConfigureStrings.self
+    static let identity = IdentityStrings.self
+    static let network = NetworkStrings.self
+    static let offering = OfferingStrings.self
+    static let purchase = PurchaseStrings.self
+    static let purchaserInfo = PurchaserInfoStrings.self
+    static let receipt = ReceiptStrings.self
+    static let restore = RestoreStrings.self
+
```

Comment:
In order not to make a lot of changes in the codebase, I left these "aliases" there. 

WDYT?

Reply:
I think leaving the aliases in this PR makes sense, and we can do another PR to remove them and change the code.